src/main/java/org/openmaptiles/layers/Water.java

```diff
@@ -201,7 +201,7 @@ void attemptNeLakeIdMapping(Tables.OsmWaterPolygon element) throws GeometryExcep
     if (!geom.isValid()) {
       geom = GeometryFixer.fix(geom);
       stats.dataError("omt_fix_water_before_ne_intersect");
-      LOGGER.debug("Fixing geometry of OSM element {} before attempt to add ID to natural earth water feature",
+      LOGGER.trace("Fixing geometry of OSM element {} before attempt to add ID to natural earth water feature",
```

Comment:
@phanecak-maptiler these are pretty verbose - any issue with me turning them to trace so they are not printed by default?

Reply:
I'll merge like this for now, feel free to move one or both back up to debug if you decide you need it.

Reply:
No problem. Thank you.